components/wmtk_components/embedded_component/internal/EmbeddedRemeshingOptions.hpp

```diff
@@ -0,0 +1,25 @@
+#pragma once
+
+#include <spdlog/spdlog.h>
+#include <nlohmann/json.hpp>
+
+namespace wmtk {
+namespace components {
+namespace internal {
+
+// this enum just for a test, we definitely should discuss the resolution design later
+enum Resolution_Level { LOW, MID, HIGH };
+
+struct EmbeddedRemeshingOptions
+{
+    std::string type;
+    std::string input;
+    std::string output;
+    Resolution_Level resolute_level;
```

Comment:
We should discuss that with Jiacheng or Daniele because I am pretty sure that they have settings for tetwild already and I guess we can copy from there.